Public/Src/Utilities/UnitTests/Utilities/HelpersTests.cs

```diff
@@ -88,5 +88,76 @@ public void RetryOnFailureAsyncTest()
 
             XAssert.IsTrue(!possibleResult.Succeeded && attemptCount == MaxAttempts);
         }
+
+        [Fact]
+        public void RetryOnExceptionTest()
+        {
+            const int MaxAttempts = 3;
+            int attemptCount = 0;
+            bool result = Helpers.RetryOnFailure(
+                lastAttempt =>
+                {
+                    attemptCount++;
+                    throw new BuildXLException(string.Empty);
+                },
+                numberOfAttempts: MaxAttempts, initialTimeoutMs: 100, postTimeoutMultiplier: 1);

```

Comment:
>initialTimeoutMs: 100 [](start = 47, length = 21)

It shouldn't matter to these tests how long is spent between attempts. Can we shrink this down to somethign much smaller to keep our test runtime small? Same comment for all other occurrences in this file